src/conversions/sample_rate.rs

```diff
@@ -47,7 +47,10 @@ where
         to: cpal::SampleRate,
         num_channels: cpal::ChannelCount,
     ) -> SampleRateConverter<I> {
-        let from = from.0;
+        let from = match from.0 {
+            0 => to.0,
```

Comment:
I would just use assertions and require at least 1 channel everywhere. Are there any use-cases you have in mind where 0 channels makes sense or can be useful?